docs/install_guides/installing-openvino-apt.md

```diff
@@ -12,6 +12,7 @@
 
    * offers both C/C++ and Python APIs
    * does not offer support for GNA and NPU inference
+   * is dedicated to Linux users only
```

Comment:
the page is already in Linux specific group. Do we need to double mention it? :)

Reply:
We mention the system so that the user doesn't have to check the menu, especially for channels supporting multiple systems. When only one system is supported, we keep this info for consistency but also adding "only" for full clarity.